apps/condo/domains/marketplace/schema/MarketCategory.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Generated by `createschema marketplace.MarketCategory 'name:Text; parentCategory:Relationship:MarketCategory:CASCADE;'`
+ */
+
+const { Relationship, File } = require('@keystonejs/fields')
+const Ajv = require('ajv')
+const addFormats = require('ajv-formats')
+
+const { GQLError } = require('@open-condo/keystone/errors')
+const { LocalizedText } = require('@open-condo/keystone/fields')
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema, getById } = require('@open-condo/keystone/schema')
+
+const { getGQLErrorValidator } = require('@condo/domains/common/schema/json.utils')
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const { getFileMetaAfterChange } = require('@condo/domains/common/utils/fileAdapter')
+const access = require('@condo/domains/marketplace/access/MarketCategory')
+const { MARKET_CATEGORY_ERRORS, ERROR_INVALID_SETTINGS } = require('@condo/domains/marketplace/constants')
+
+const MARKET_CATEGORY_FILE_ADAPTER = new FileAdapter('market_category')
+const imageMetaAfterChange = getFileMetaAfterChange(MARKET_CATEGORY_FILE_ADAPTER, 'image')
+
+const ajv = new Ajv()
+addFormats(ajv)
+
+const settingsFieldSchema = {
+    type: 'object',
+    additionalProperties: false,
+    required: ['bgColor', 'titleColor'],
+    properties: {
+        bgColor: {
+            type: 'string',
+            pattern: '^#(?:[0-9a-fA-F]{3}){1,2}$',
+        },
+        titleColor: {
+            type: 'string',
+            pattern: '^#(?:[0-9a-fA-F]{3}){1,2}$',
+        },
+    },
+}
+const validateSettingsField = getGQLErrorValidator(ajv.compile(settingsFieldSchema), ERROR_INVALID_SETTINGS)
+
+const MarketCategory = new GQLListSchema('MarketCategory', {
+    schemaDoc: 'An entity that allows you to define a category tree in the marketplace',
+    adminDoc: 'mobileSettings={bgColor:’#rrggbb’, titleColor:’#rrggbb’}',
```

Comment:
It's better to write this in ready to copy-paste manner
```suggestion
    adminDoc: 'mobileSettings = {"bgColor":"#rrggbb", "titleColor":"#rrggbb"}',
```